ifrs17/Import/Importers.ipynb

```diff
@@ -503,18 +503,33 @@
         {
             "cell_type": "code",
             "source": [
-                "public async Task<ImportArgs[]> GetAllArgsAsync(ImportArgs args, IDataSource dataSource, string format, IDataSource targetDataSource)",
+                "public async Task<ImportArgs[]> GetAllArgsAsync(ImportArgs args, IDataSource dataSource, string format)",
                 "\n{",
                 "\n    ImportArgs[] allArgs;",
                 "\n    switch(format)",
                 "\n    {",
-                "\n        case ImportFormats.YieldCurve : ",
+                "\n        case ImportFormats.Cashflow:",
+                "\n        case ImportFormats.Actual: ",
+                "\n        case ImportFormats.Opening:",
+                "\n        case ImportFormats.SimpleValue:",
```

Comment:
why SimpleValue is here? this format does not compute anything.

Reply:
Good point.

Reply:
yes, agree, we can remove simple value.